apps/src/StudioApp.js

```diff
@@ -93,7 +93,7 @@ var copyrightStrings;
 /**
  * The minimum width of a playable whole blockly game.
  */
-const MIN_WIDTH = 1200;
+const MIN_WIDTH = 1600;
```

Comment:
This will affect all tutorials.. do we know whether enough of them were affected that it's worth changing the zoom factor on all of them?

Reply:
hmm I'm not sure. I am having a hard time finding another one that has a finish button. @moneppo do we want our mobile view to be optimized for dance party, and be zoomed out (potentially unnecessarily) for other labs on mobile?

Reply:
found some other finish button ones, they all seem to work fine with the current zoom level. So it's a question of whether we want to optimize for dance party, find a way to only apply this to dance party, or just leave dance party as-is

Reply:
potential middle ground (although it may work less well for dance party across many devices) is to set this to `1400`, which will let tests pass but won't be quite so zoomed out everywhere. Here's the screenshot for dance party (this is from saucelabs so I don't know how to make it less blurry)
<img width="932" alt="Screen Shot 2023-01-06 at 10 43 09 AM" src="https://user-images.githubusercontent.com/33666587/211078100-5961979b-d16f-4971-951a-a91ba374d58a.png">